### PR TITLE
added_a_blog

### PR DESCRIPTION
**Pull request by codewithkushagra**
### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?

### New Feature Submissions:

1. [x] Does your submission pass tests?
2. [x] Have you lint your code locally prior to submission?

### Changes to Core Features:

* [ ] Have you added an explanation of what your changes do and why you'd like us to include them?
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: _pages/author-kushagra.html [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: "kushagra Agarwal"
```
Comment (AvikantSrivastava) [inline _pages/author-kushagra.html]: Make "K" capital

File: _config.yml [hunk 2]
```diff
@@ -119,6 +119,13 @@ authors:
     email:          vst0010@gmail.com
     twitter:        https://www.instagram.com/vst_10/
 
+  kushagra:
+    name:           Kushagra Agarwal
+    site:           https://codewithkushagra.netlify.app/
+    avatar:         "assets/images/authors/kushagra.jpg"
+    bio:            "Am a code geek, a NARUTO fan and also love expressing my depression in words :)"
```
Comment (AvikantSrivastava) [inline _config.yml]: It should be "I am"

File: Gemfile.lock [hunk 3]
```diff
@@ -4,65 +4,69 @@ GEM
     addressable (2.7.0)
       public_suffix (>= 2.0.2, < 5.0)
     colorator (1.1.0)
-    concurrent-ruby (1.1.6)
```
Comment (AvikantSrivastava) [inline Gemfile.lock]: avoid making changes to this file

**Review by AvikantSrivastava — CHANGES_REQUESTED** [review]
Please follow my comments on the `Gemfile.lock` and `_pages/author-kushagra.html`

`

File: _pages/author-kushagra.html (hunk 1 above)
Comment (codewithkushagra) [reply _pages/author-kushagra.html]: oh, okay I will just change that

**Review by AvikantSrivastava — CHANGES_REQUESTED** [review]
@codewithkushagra please avoid, changing the `gemfile` 
image

File: Gemfile.lock (hunk 3 above)
Comment (codewithkushagra) [reply Gemfile.lock]: oh, okay 
should I remove this change 
actually that version of ruby was not working for me. so, I changed locally but I pushed it too by mistake

Comment (codewithkushagra) [reply Gemfile.lock]: @AvikantSrivastava  
I have made all the changes 
can you review my code again

**Review by AvikantSrivastava — APPROVED** [review]
Amazing the changes look perfect, Thanks for contributing to [LEO MVIT](https://github.com/leomvit/)

**Pull request merged**